docs/feature_rawhid.md

```diff
@@ -0,0 +1,119 @@
+# Raw HID
+
+Raw HID allows for bidirectional communication between keyboard and a host computer over HID interface. It allows for 32 byte bidirectional packet transfer in QMK.
```

Comment:
```suggestion
Raw HID allows for bidirectional communication between QMK and the host computer over an HID interface. This has many potential use cases, such as switching keymaps on the fly or changing RGB LED colors and modes.
```

The goal for Raw HID seems to be to implement variable length packets up to 64 bytes, so I think the current limitations should go instead in a caveats section.

Reply:
Good to see that's on the books. I was wondering about that.